New version: IceFloeTracker v0.6.0

JuliaRegistrator opened the pull request:
- Registering package: IceFloeTracker
- Repository: https://github.com/WilhelmusLab/IceFloeTracker.jl
- Created by: @cpaniaguam
- Version: v0.6.0
- Commit: b77ec56ef3aea73a75a3a8033ec5e47614e393c8
- Reviewed by: @cpaniaguam
- Reference: https://github.com/WilhelmusLab/IceFloeTracker.jl/commit/b77ec56ef3aea73a75a3a8033ec5e47614e393c8#commitcomment-148379795
- Description: Julia package for ice floe tracker
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6df8ea0f25044b7ca9203f5c12bc474acf94a54b69d05a1726e7c093df503ec8e37ae13c0ef29781c23aa31291abec42d3d79e37c0d365a6c6ab559ffadee93ddd818b8f44ec00d27d1b728f22e218da33f1b080e96f0bf9f293fc698363785f1dc11dd8ad7e5034b79370d582d80319d5a80f8a802d2862ecb5269f64c6743f453d06932bcf2a9f2dde225a929fdd95f4913acb72a40163ca78bac1c7f43f34513e112c4ce17514da705f15c33b21486 -->

cpaniaguam commented:
All but the AutoMerge action fails due to use of Python 3.12 by the CI which is not compatible with the versions of our Python deps. Is there a way to change that to Python 3.11? If not, please merge manually. Thanks!

lgoettgens commented:
> The following dependencies do not have a [compat] entry that is upper-bounded and only includes a finite number of breaking releases: ImageBinarization, TiledIteration

This has to be fixed by you

[noblock]

cpaniaguam commented:
> > The following dependencies do not have a [compat] entry that is upper-bounded and only includes a finite number of breaking releases: ImageBinarization, TiledIteration
> 
> This has to be fixed by you
> 
> [noblock]

@lgoettgens That has been fixed. Please see 
https://github.com/WilhelmusLab/IceFloeTracker.jl/blob/ede819aa48550e70484cf45c56dd6fa2cbb5b66d/Project.toml#L71
https://github.com/WilhelmusLab/IceFloeTracker.jl/blob/ede819aa48550e70484cf45c56dd6fa2cbb5b66d/Project.toml#L97

cpaniaguam commented:
> > The following dependencies do not have a [compat] entry that is upper-bounded and only includes a finite number of breaking releases: ImageBinarization, TiledIteration
> 
> This has to be fixed by you
> 
> [noblock]

@lgoettgens That has been fixed. They werent listed in the compat section before but now they are. Please see 
https://github.com/WilhelmusLab/IceFloeTracker.jl/blob/ede819aa48550e70484cf45c56dd6fa2cbb5b66d/Project.toml#L71
https://github.com/WilhelmusLab/IceFloeTracker.jl/blob/ede819aa48550e70484cf45c56dd6fa2cbb5b66d/Project.toml#L97

Do they need something like "<2"? The other ones don't seem to need that.

lgoettgens commented:
"0" isn't a valid compat string. See the pkg.jl documentation for details. But basically, you have to list all `0.x` versions that you are compatible with (since all of these are considered pair wise breaking)

GunnarFarneback commented:
Tip: https://github.com/GunnarFarneback/PackageCompatUI.jl can help you set your compat.

cpaniaguam commented:
> Tip: https://github.com/GunnarFarneback/PackageCompatUI.jl can help you set your compat.

@GunnarFarneback @lgoettgens 
I think this is good now. Could you please take another look? Thanks for guidance!

JuliaRegistrator committed:
New version: IceFloeTracker v0.6.0

UUID: 04643c7a-9ac6-48c5-822f-2704f9e70bd3
Repo: https://github.com/WilhelmusLab/IceFloeTracker.jl.git
Tree: f3485b68094eb7a683c080c5a48ea35abdf7a1f4

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

ericphanson committed:
Merge branch 'master' into registrator-icefloetracker-04643c7a-v0.6.0-bf006c3da6

lgoettgens commented:
[noblock]

a lot of comments above are missing the [noblock] tag

And of course there is still the problem with loading the package
